src/components/OrderItems.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react";
+
+class OrderItems extends Component {
+  render() {
+    return (
+      <select onChange={this.props.onSortToggle}>
+        <option value={true}>Ordenar</option>
+        <option value={false}>Não Ordenar</option>
+      </select>
+    );
+  }
+}
```

Comment:
Se esse componente não tem estado nem métodos, que tal transformá-lo em uma simples função?

```suggestion
function OrderItems(props) {
    return (
      <select onChange={props.onSortToggle}>
        <option value={true}>Ordenar</option>
        <option value={false}>Não Ordenar</option>
      </select>
    );
}
```